.github/workflows/test.yaml

```diff
@@ -92,10 +92,10 @@ jobs:
         run: make tidy && git diff -s --exit-code || (echo "make tidy needed"; exit 1)
       - name: Check gofmt
         run: make fmt  && git diff -s --exit-code || (echo  "make fmt needed"; exit 1)
-      - name: Check licenses file
```

Comment:
Please revert

Reply:
Yeah, was just checking the `fmt` stuff was 👍